### PR TITLE
Fix getContainer() return type

### PR DESCRIPTION
**Pull request by cjh1**
### Changes

- [x] Documentation and TypeScript definitions were updated to match those changes

### PR and Code Checklist

- [x] [semantic-release](https://github.com/semantic-release/semantic-release) commit messages
- [x] Run `npm run reformat` to have correctly formatted code

### Testing

- [x] This change adds or fixes unit tests 
- [x] Tested environment:
  - **vtk.js**: 
  - **OS**: 
  - **Browser**:

### CODE REVIEW

File: Sources/Rendering/OpenGL/RenderWindow/index.d.ts [hunk 1]
```diff
@@ -66,7 +66,7 @@ export interface vtkOpenGLRenderWindow extends vtkOpenGLRenderWindowBase {
 	/**
 	 * Get the container element.
 	 */
-	getContainer(): void;
+	getContainer(): HTMLElement;
```
Comment (floryst) [inline Sources/Rendering/OpenGL/RenderWindow/index.d.ts:69]: The container can be null.

Comment (cjh1) [reply Sources/Rendering/OpenGL/RenderWindow/index.d.ts]: Fixed

**Comment by daker** [comment]
Can you change the commit message to : docs(ts): fix vtkOpenGLRenderWindow defs

File: Sources/Rendering/OpenGL/RenderWindow/index.d.ts (hunk 1 above)
Comment (daker) [reply Sources/Rendering/OpenGL/RenderWindow/index.d.ts]: You can use Nullable from [types.d.ts](https://github.com/Kitware/vtk-js/blob/master/Sources/types.d.ts#L4)
`Nullable<HTMLElement>`

Comment (cjh1) [reply Sources/Rendering/OpenGL/RenderWindow/index.d.ts]: Great, thanks

**Review by daker — APPROVED** [review]
LGTM

**Pull request merged**